installer/pkg/config-generator/generator.go

```diff
@@ -13,7 +13,6 @@ import (
 	"github.com/coreos/tectonic-config/config/kube-addon"
 	"github.com/coreos/tectonic-config/config/kube-core"
 	"github.com/coreos/tectonic-config/config/tectonic-network"
-	tnco "github.com/coreos/tectonic-config/config/tectonic-node-controller"
```

Comment:
You probably want a separate commit rebuilding `BUILD.bazel` and cleaning this our of `vendor/`.

Reply:
> I didn't remove glide.lock, as i didn't want to update the whole world :/

Removing `glide.lock` wasn't too bad when I did it in 2018-08-30 in 2e835b0bd.  But I'm fine punting the `vendor/` cleanup down the road if you want.